pression/pluginRegistry.cpp

```diff
@@ -59,8 +59,11 @@ class PluginRegistry
             directories.push_back( getcwd( cwd, MAXPATHLEN ));
 
 #ifdef _WIN32
+            //rgh: does not compile, and seems to be handled by code above.
+            /*
             if( GetModuleFileName( 0, cwd, MAXPATHLEN ) > 0 )
                 directories.push_back( pression::getDirname( cwd ));
```

Comment:
Ok, I understand.
With these changes, I can compile equalizer in windows.
All the eq* demos work, but the seqPly shows a black screen (may be because of my commented lines). I will add your changes and retry.